tutorials/xr/a_better_xr_start_script.rst

```diff
@@ -91,7 +91,16 @@ We introduce a few new variables to our script as well:
 Our updated ready function
 --------------------------
 
-The ready function mostly remains the same but we hook up a number of signals that will be emitted by the :ref:`XRInterface <class_xrinterface>`.
+We add a few things to the ready function.
+
+If we're using the mobile or forward+ renderer we set the viewports ``vrs_mode`` to ``VRS_XR``.
+On platforms that support this, this will enable foveated rendering.
```

Comment:
Would a platform in this case be referring to the headset or the OpenXR runtime(SteamVR, Mixed Reality Portal)?

Reply:
In this case it's unrelated to the XR side, VRS is implemented in Vulkan so it's up to the capabilities of the GPU driver.